src/app/view/perspectives/pivot/pipe/pivot-clear-attribute.pipe.ts

```diff
@@ -0,0 +1,21 @@
+import {Pipe, PipeTransform} from '@angular/core';
+import {PivotAttribute} from '../../../../core/store/pivots/pivot';
+
+@Pipe({
+  name: 'pivotClearAttribute'
+})
+export class PivotClearAttributePipe implements PipeTransform {
+
+  public transform(attribute: PivotAttribute): PivotAttribute {
+    if (!attribute) {
+      return attribute;
+    }
+    return {
+      attributeId: attribute.attributeId,
+      resourceIndex: attribute.resourceIndex,
+      resourceId: attribute.resourceId,
+      resourceType: attribute.resourceType
```

Comment:
Because attributes contains some additional information and `select-item.component` do some kind of `equals` on it. But i will use the function from `pivot-util.ts`.